Frameworks/GLKit/GLKMath.mm

```diff
@@ -674,13 +674,10 @@ GLKIT_EXPORT GLKVector3 GLKMatrix4MultiplyVector3(GLKMatrix4 m, GLKVector3 vec)
    @Status Interoperable
 */
 GLKIT_EXPORT GLKVector3 GLKMatrix3MultiplyVector3(GLKMatrix3 m, GLKVector3 vec) {
-    GLKVector3 res;
-
-    res.x = m.m00 * vec.x + m.m10 * vec.y + m.m20 * vec.z;
-    res.y = m.m01 * vec.x + m.m11 * vec.y + m.m21 * vec.z;
-    res.z = m.m02 * vec.x + m.m12 * vec.y + m.m22 * vec.z;
-
-    return res;
+    GLKVector3 v = { m.m[0] * vec.v[0] + m.m[3] * vec.v[1] + m.m[6] * vec.v[2],
```

Comment:
I think the previous version was more readable.  Are you making this change to fix a bug?  If so, please add a test for this method.